controllers/controllers.js

```diff
@@ -30,13 +30,33 @@ exports.createUser = (req, res) => {
     })
     .catch(err => {
       res.status(500).send({
-        message: err.message || "Something broke while creating the product"
+        message: "Username already taken" 
```

Comment:
It could be a database connection failure as well. Could you tell the difference between these cases?